src/schema/v2/sale_artwork.ts

```diff
@@ -318,26 +318,39 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       calculatedCost: {
-        type: CalculatedCostType,
+        type: CalculatedCost,
         args: {
-          bidAmountCents: {
-            type: GraphQLInt,
+          bidAmountMinor: {
+            type: new GraphQLNonNull(GraphQLInt),
             description: "Max bid price for the sale artwork",
           },
         },
-        resolve: (_params, { bidAmountCents }, _loaders) => {
+        resolve: async (
+          { id, sale_id },
+          { bidAmountMinor },
+          { saleArtworkCalculatedCostLoader }
+        ) => {
+          const data = await saleArtworkCalculatedCostLoader({
+            saleId: sale_id,
+            saleArtworkId: id,
+            bidAmountMinor,
+          })
+
           return {
             buyersPremium: {
-              cents: bidAmountCents * 0.2,
-              display: `$${((bidAmountCents * 0.2) / 100)
-                .toFixed(2)
-                .replace(/\d(?=(\d{3})+\.)/g, "$&,")}`,
+              cents: data.buyers_premium_cents,
```

Comment:
This seems a bit surprising to me. How is it that the tests assert upon the API breaking change of cents/dollars -> minor/major yet we're still returning cents here? Is there some additional layer of data conversation to major/minor?

Reply:
Ah perhaps this seems to be something to due with the `Money` type expecting `cents` as input. Bit mysterious to me perhaps, probably this is the pattern 👍

Reply:
Yep, that's right. The Money type handles that conversion.